.travis/template.txt

```diff
@@ -15,7 +15,7 @@ before_script:
 - cd build 
 - cmake -DCGAL_HEADER_ONLY=ON -DQt5_DIR="/opt/qt55/lib/cmake/Qt5" -DQt5Svg_DIR="/opt/qt55/lib/cmake/Qt5Svg" -DQt5OpenGL_DIR="/opt/qt55/lib/cmake/Qt5OpenGL" -DWITH_demos:BOOL=TRUE -DWITH_examples:BOOL=true -DWITH_tests:BOOL=TRUE -DCMAKE_CXX_FLAGS_RELEASE=-DCGAL_NDEBUG .. 
 - make 
-- sudo make install
+- sudo make install --silent
```

Comment:
Have you tried locally? I think the output is made by the commands called my `make`, and not by `make` itself.

Reply:
Indeed.